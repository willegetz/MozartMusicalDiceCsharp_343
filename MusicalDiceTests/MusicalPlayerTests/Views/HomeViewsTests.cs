using ApprovalTests.Asp;
using ApprovalTests.Asp.Mvc;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using MusicalDiceTests.MusicalPlayerTests.TestableControllers;
using MusicalDiceTests.MusicalPlayerTests.Utilities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MusicalDiceTests.MusicalPlayerTests.Views
{
    // TODO: Start Without Debugging to launch IISExpress and be able to use the MvcApprovals.

    [TestClass]
    [UseReporter(typeof(BeyondCompare4Reporter))]
    public class HomeViewsTests
    {
        [TestMethod]
        public void IndexView()
        {
            PortFactory.MvcPort = 49300;
            MvcApprovals.VerifyMvcPage<TestableHomeController>(home => home.TestIndex, CustomMvcScrubbers.CopyrightScrubber);
        }
    }
}
