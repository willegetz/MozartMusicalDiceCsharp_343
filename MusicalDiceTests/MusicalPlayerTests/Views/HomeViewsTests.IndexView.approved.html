<!DOCTYPE html>
<html>
<head><base href="http://localhost:49300/TestableHome">
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Mozart Musica Dice Player</title>
    <link href="/Content/bootstrap.css" rel="stylesheet"/>
<link href="/Content/site.css" rel="stylesheet"/>

    <script src="/Scripts/modernizr-2.6.2.js"></script>

</head>
<body>
    <div class="navbar navbar-inverse navbar-fixed-top">
        <div class="container">
            <div class="navbar-collapse collapse">
                <ul class="nav navbar-nav">
                    <li><a href="/">Home</a></li>
                    <li><a href="/Home/About">About</a></li>
                    <li><a href="/Home/Contact">Contact</a></li>
                </ul>
            </div>
        </div>
    </div>
    <div class="container body-content">
        


<div class="jumbotron">
    <h1>Mozart Musical Dice Player</h1>
    <p class="lead">My solution for the <a href="https://www.reddit.com/r/dailyprogrammer/comments/7i1ib1/20171206_challenge_343_intermediate_mozarts/" target="_blank">Mozart Musical Dice</a> challenge from Reddit's <a href="https://www.reddit.com/r/dailyprogrammer/" target="_blank">Daily Programmer</a> subreddit.</p>
</div>

<div class="row">
    <div class="text-center">
        <p></p>
    </div>
</div>
        <hr />
        <footer>
            <p>&copy; 0000 - Mozart Musical Dice Player</p>
        </footer>
    </div>

    <script src="/Scripts/jquery-1.10.2.js"></script>

    <script src="/Scripts/bootstrap.js"></script>
<script src="/Scripts/respond.js"></script>

    
</body>
</html>
